backend/api/appsettings.Development.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
     }
+  },
+  "ConnectionStrings": {
+    "PIMS": "Server=pims-db-dev,1433;User ID=sa;Database=geospatial"
```

Comment:
If we are going to hard-code a default, I would suggest that we use the local instance as set out in `docker-compose.yaml`. That why for cloud deployments (ie Openshift) you are required to overwrite the default with proper ENV vars

Reply:
Generally I use these files for environments rather than local development.  But it can go either way.